### PR TITLE
Project Hermes blogpost

### PR DESCRIPTION
**Pull request by da-kami**
I decided to write this blogpost first (to be cross-posted on comit-network Medium). 
This post should help our community understand what is going on. We can link it where needed.

---

On top of that I'd write the public testnet demo blogpost that should contain:

- instructions on ho to use the binary and the docker container (just one command)
- screenrecorded video that shows the usage against our public testnet maker

### CODE REVIEW

File: blog/2021-11-08-hermes-goes-itchysats.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+tags: [bitcoin,defi,liquid,options,futures,loans]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img alt="Hermes" src={useBaseUrl('blog/assets/images/2021-11/hermes-goes-itchysats.png')} />
+
+In this blogpost we introduce you to our latest project: Hermes, CFD trading using DLCs on Bitcoin.
+Other than project baru we decided to build on Bitcoin directly rather than building on the Liquid sidechain.
+We are teaming up with [ItchySats](https://www.itchysats.network/) for CFD trading directly on Bitcoin and support them with DLC research.
```
Comment (bonomat) [inline blog/2021-11-08-hermes-goes-itchysats.md:15]: Love it ❤️

File: blog/2021-11-08-hermes-goes-itchysats.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+DeFi - short for decentralised finance - can mean many things.
+In the Ethereum world, DeFi is all about smart contracts with governance tokens, yield farming, AMM pools, collateralised loans and lots of other stuff.
+Ethereum's smart contract model makes building these things reasonable easy and the pace of innovation is quite incredible.
+
+In the Bitcoin world, many consider Bitcoin to be the original DeFi.
+Bitcoin means [many things](https://medium.com/@nic__carter/visions-of-bitcoin-4b7b7cbcd24c) to many different people.
+If we think about it as a financial asset, it makes sense to build financial products on top of it.
+We think Bitcoin is the world's best leverage - time to leverage it!
+
+Of course, while doing so, we don't want to compromise Bitcoin's core values: decentralised, censorship-resistant and permission-less.
```
Comment (bonomat) [inline blog/2021-11-08-hermes-goes-itchysats.md:30]: Imho this is one word. 

```suggestion
Of course, while doing so, we don't want to compromise Bitcoin's core values: decentralised, censorship-resistant and permissionless.
```

File: blog/2021-11-08-hermes-goes-itchysats.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Bitcoin is constantly evolving.
+As you would expect from a system that governs [billions of dollars](https://coinmarketcap.com/), it moves at a fairly slow rate.
+
+While working on project baru, our Bitcoin loan solution on the Liquid sidechain, the ItchySats team approached us with one question:
+
+> Is there any derivative that we *can* build directly on present day Bitcoin?
+
+Yes there is! We can actually achieve CFD (Contract For Difference) trading.
+Using DLCs and an Oracle we are able to use the world's best leverage directly on layer 1!
```
Comment (bonomat) [inline blog/2021-11-08-hermes-goes-itchysats.md:42]: I think you mean collateral right? 

```suggestion
Using DLCs and an Oracle we are able to use the world's best collateral directly on layer 1!
```

**Review by bonomat — APPROVED** [review]
You are a photoshop wizard :D 

Great writeup

File: blog/2021-11-08-hermes-goes-itchysats.md (hunk 3 above)
Comment (da-kami) [reply blog/2021-11-08-hermes-goes-itchysats.md:42]: riiight, my brain was not working 😅

**Pull request merged**